src/content/docs/data-apis/custom-data/aws-privatelink.mdx

```diff
@@ -322,6 +336,11 @@ These are the New Relic endpoint services available via AWS PrivateLink:
   </TabsPages>
 </Tabs>
 
+<Callout variant="important">
+  Review the following constraints when configuring the `identity-api.newrelic.com` and `infrastructure-command-api.newrelic.com` endpoints:
+  - These are only exposed in the `us-east-2` (Ohio) region.
+  - The endpoint service does not have DNS Private name associated. Create a PrivateLink connected to this Service Endpoint for each API, and create the Private Hosted Zone (PHZ) for each API Domain. 
```

Comment:
Thanks for reviewing this:
1. Sure, updated. 
2. I adapted this to align with the terminology in the table headers: hostnames and service endpoints. 

Also, I moved the callout within the US table only since these are only available in US.